app/(root)/layout.tsx

```diff
@@ -1,12 +1,12 @@
 import './globals.css';
 
 import { Analytics } from '@vercel/analytics/react';
-import { SessionProvider } from 'next-auth/react';
-import { SpeedInsights } from "@vercel/speed-insights/next"
+import { SessionProvider } from '@/components/component/authProvider';
```

Comment:
Weird operation

Reply:
you have to make it a client component for more context checkout https://stackoverflow.com/questions/75902311/next-js-13-and-next-auth-issues-with-usesession-and-sessionprovider